ingest-app/src/cmr/ingest/services/event_handler.clj

```diff
@@ -32,11 +32,20 @@
 (defmethod handle-provider-event :default
   [_ _])
 
+(defn- handle-gran-bulk-update-event
```

Comment:
Should move this code to the branch that does the implementation instead of introducing code that doesn't do anything

Reply:
I'm going to use this immediately so one way or another it's not going to be sitting

Reply:
Ok,

Reply:
This function definition cannot be removed. It is used to set up event handler for the queue. See `subscribe-to-events` function.